dockerfiles/fastapi/Dockerfile

```diff
@@ -4,7 +4,7 @@ COPY app /app
 
 WORKDIR /app
 
-RUN cd /app && pip install -r requirements.txt
+RUN cd /app && pip install -r requirements.txt && export $(grep -v '^#' /app/.env | xargs)
```

Comment:
What does this do?

Reply:
don't know where that is coming from either

Reply:
Alright I remember now where this is from, but not sure how that ended up in the commit. That was from the other ticket about extracting all env vars, but this bit never got merged. I'll remove it